### ISSUE TITLE
Arguments for call are not valid.

### ISSUE THREAD

**Issue by wuyiulin** [diegovalsesia/HyCoRe#3]
Hi, @diegovalsesia ,

Thanks for releasing the package. When I run the command python main_pointmlp_hycore.py --msg=Offv_pointmlp_hycore_var --workers=8, I got the following error:

**Arguments for call are not valid.
The following variants are available:
  
  aten::tensordot(Tensor self, Tensor other, int[] dims_self, int[] dims_other) -> (Tensor):
  Expected a value of type 'List[int]' for argument 'dims_self' but instead found type 'Tuple[List[int], List[int]]'.
  
  aten::tensordot.out(Tensor self, Tensor other, int[] dims_self, int[] dims_other, *, Tensor(a!) out) -> (Tensor(a!)):
  Expected a value of type 'List[int]' for argument 'dims_self' but instead found type 'Tuple[List[int], List[int]]'.

The original call is:
  File "/home/lab715-3090-03/anaconda3/envs/hycore/lib/python3.7/site-packages/geoopt/manifolds/stereographic/math.py", line 1238
    x_norm = x.norm(dim=dim, keepdim=True, p=2).clamp_min(1e-15)
    if dim != -1 or m.dim() == 2:
        mx = torch.tensordot(x, m, ([dim], [1]))
             ~~~~~~~~~~~~~~~ <--- HERE
    else:
        mx = torch.matmul(m, x.unsqueeze(-1)).squeeze(-1)**


When i run `sh launcher_pointmlp.sh`, got the same error.

It looks like dataset path error?

Any hints to fix this issue?

Thanks~

**Comment by wuyiulin** [comment]
Answer by my self, you should use pip install geoopt==0.3

because pip will default install 0.5.0 for u,
use 0.3 version can run this command `python main_pointmlp_hycore.py --msg=Offv_pointmlp_hycore_var --workers=8`
